### ISSUE TITLE
[iOS] NSRangeException - cellForItemAtIndexPath

### ISSUE THREAD

**Issue by cjpete** [tinycreative/react-native-intercom#408]
Crash picked up via Sentry, so unclear on the exact means to replicate, but posting incase anyone else has seen the same issue.

```
*** -[__NSArrayM objectAtIndexedSubscript:]: index 7 beyond bounds [0 .. 6]
```

![image](https://user-images.githubusercontent.com/13134961/99146590-820d4600-2671-11eb-9b56-4045953acf0f.png)

Using react-native-intercom@18.0.0 and Intercom pod @ 8.0.0

**Comment by cjpete** [comment]
I can confirm this has also been seen on versions prior to `18.0.0`.

**Comment by Br1an-Boyle** [comment]
Please update to the latest version that uses v9.0.0 of the Intercom SDK https://github.com/intercom/intercom-ios/releases/tag/9.0.0. This issue was fixed in [8.1.1](https://github.com/intercom/intercom-ios/releases/tag/8.1.1) but I recommend updating to 9.0.0.
This will fix the problem
